src/navsat_transform.cpp

```diff
@@ -273,34 +276,38 @@ namespace RobotLocalization
       tf2::Quaternion imu_quat;
       imu_quat.setRPY(0.0, 0.0, imu_yaw);
 
-      // The transform order will be orig_odom_pos * orig_utm_pos_inverse * cur_utm_pos.
+      // The transform order will be orig_odom_pos * orig_cartesian_pos_inverse * cur_cartesian_pos.
       // Doing it this way will allow us to cope with having non-zero odometry position
       // when we get our first GPS message.
-      tf2::Transform utm_pose_with_orientation;
-      utm_pose_with_orientation.setOrigin(transform_utm_pose_corrected.getOrigin());
-      utm_pose_with_orientation.setRotation(imu_quat);
+      tf2::Transform cartesian_pose_with_orientation;
+      cartesian_pose_with_orientation.setOrigin(transform_cartesian_pose_corrected.getOrigin());
+      cartesian_pose_with_orientation.setRotation(imu_quat);
 
-      utm_world_transform_.mult(transform_world_pose_, utm_pose_with_orientation.inverse());
+      cartesian_world_transform_.mult(transform_world_pose_, cartesian_pose_with_orientation.inverse());
 
-      utm_world_trans_inverse_ = utm_world_transform_.inverse();
+      cartesian_world_trans_inverse_ = cartesian_world_transform_.inverse();
 
       ROS_INFO_STREAM("Transform world frame pose is: " << transform_world_pose_);
-      ROS_INFO_STREAM("World frame->utm transform is " << utm_world_transform_);
+      ROS_INFO_STREAM("World frame->cartesian transform is " << cartesian_world_transform_);
 
       transform_good_ = true;
 
-      // Send out the (static) UTM transform in case anyone else would like to use it.
-      if (broadcast_utm_transform_)
+      // Send out the (static) Cartesian transform in case anyone else would like to use it.
+      if (broadcast_cartesian_transform_)
       {
-        geometry_msgs::TransformStamped utm_transform_stamped;
-        utm_transform_stamped.header.stamp = ros::Time::now();
-        utm_transform_stamped.header.frame_id = (broadcast_utm_transform_as_parent_frame_ ? "utm" : world_frame_id_);
-        utm_transform_stamped.child_frame_id = (broadcast_utm_transform_as_parent_frame_ ? world_frame_id_ : "utm");
-        utm_transform_stamped.transform = (broadcast_utm_transform_as_parent_frame_ ?
-                                             tf2::toMsg(utm_world_trans_inverse_) : tf2::toMsg(utm_world_transform_));
-        utm_transform_stamped.transform.translation.z = (zero_altitude_ ?
-                                                           0.0 : utm_transform_stamped.transform.translation.z);
-        utm_broadcaster_.sendTransform(utm_transform_stamped);
+        geometry_msgs::TransformStamped cartesian_transform_stamped;
+        cartesian_transform_stamped.header.stamp = ros::Time::now();
+        std::string cartesian_frame_id = (use_local_cartesian_)? "local_enu" : "utm";
```

Comment:
The parentheses here are a little off. To retain the original style, we should do this:

`std::string cartesian_frame_id = (use_local_cartesian_? "local_enu" : "utm");`

I realize the parentheses are unnecessary, but IMO it makes these things more readable.

Reply:
Done.